justfile

```diff
@@ -3,7 +3,7 @@ just := just_executable()
 _build:
```

Comment:
Please lets just rename this to `build` then, given we should separate these two steps.

Reply:
Done!